go-controller/pkg/config/config.go

```diff
@@ -180,7 +179,7 @@ type CNIConfig struct {
 	ConfDir string `gcfg:"conf-dir"`
 	// Plugin specifies the name of the CNI plugin
 	Plugin string `gcfg:"plugin"`
-	// Windows ONLY, specifies the ID of the HNS Network to which the containers will be attached
+	// UNUSED; remove in the near future
 	WinHNSNetworkID string `gcfg:"win-hnsnetwork-id"`
```

Comment:
Is there any reason to keep this? No one on Linux should be trying to set it

Reply:
Yeah.